.circleci/config.yml

```diff
@@ -77,3 +77,11 @@ workflows:
                 - 3.1.4
                 - 3.2.2
                 - 3.3.0
+      - test_modern:
```

Comment:
CI is failing because the `test_modern` job isn't defined. Can you instead add the `gemfiles/avro1_11_rails7_2.gemfile` Gemfile to the above build matrix and use appropriate [exclusions](https://circleci.com/docs/configuration-reference/#excluding-sets-of-parameters-from-a-matrix) for unsupported versions of Ruby? Here are some [examples](https://github.com/salsify/goldiloader/blob/master/.circleci/config.yml#L92) from other projects.

Reply:
Ah gotcha. Thanks for explaining that.